docs/GettingStartedDocs/Windows_vscode.md

```diff
@@ -0,0 +1,118 @@
+# Building And Debugging Using Visual Studio Code
+
+This document provides a brief overview of how to build and debug Open Enclave applications using VS Code.
+
+## Install VS Code
+
+The latest version of Visual Studio Code can be installed from [https://code.visualstudio.com/](https://code.visualstudio.com/)
+
+
+## Install VS Code Extensions
+
+Install the following VS Code extensions. Click on an image to navigate to the Visual Studio Code Marketplace page for the extension.
+
+[![C/C++ Extension](images/VSCodeCppExtension.png)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)
+
+[![CMake Tools Extension](images/VSCodeCMakeToolsExtension.png)](https://marketplace.visualstudio.com/items?itemName=vector-of-bool.cmake-tools)
+
+[![CDB Extension](images/VSCodeCDBExtension.png)](https://marketplace.visualstudio.com/items?itemName=MicrosoftDebuggingPlatform.vscode-cdb)
+
+## Launch Visual Studio Code
+Open an instance of x64 Native Tools Command Prompt. This ensures that cmake is available in the path.
+
+![x64 Native Tools Command Prompt](images/VSCodeNativeToolsPrompt.png)
+
+Change to the directory containing your Open Enclave Application and launch VS Code in the current folder.
+
+```cmd
+cd YourApplicationFolder
+code .
+```
+
+![Launch VS Code](images/VSCodeLaunch.png)
+
+## Configure Your Workspace
+
+Open Workspace Settings by pressing Ctrl+Shift+P and typing "Open Workspace Settings" in the command palette.
+
+![Open Workspace Settings](images/VSCodeOpenWorkspaceSettings.png)
+
+Search for the setting "CMake Configure Args" and add an item `-DNUGET_PACKAGE_PATH=path-to-openenclave-nuget-packages\prereqs\nuget`.
```

Comment:
Do we still need the `prereqs\nuget` part in the path?

Reply:
At this moment, yes. @johnkord is fixing this.